workshops/static/bulk_email.js

```diff
@@ -0,0 +1,27 @@
+/*
+FIXME I'm not JS guru and I've no idea what is The Right Way to export
+a function. I guess not the way I did it, because this way pollutes global
+space.
```

Comment:
Looks good to me -- but I'm not JS expert either (never heard anyone developing AMY call themselves a JS expert :-) ).